views/404.view.php

```diff
@@ -1,3 +1,20 @@
 <?php include('partials/header.view.php') ?>
-    <p class="text-5xl text-center md:mt-32"><span class="text-blue-700 md:text-6xl font-bold">404</span> <br> This page was not found</p>
+    <p class="text-5xl text-center md:mt-32">
+        <span class="text-blue-700 md:text-6xl font-bold">404</span> 
+        <br> 
+        
+        <?php 
+            if(!empty($templateVars['error'])) {
```

Comment:
Since you're calling `extract($templateVars)` inside `view()`, I believe this should work also:

 ```suggestion
            if(!empty($error)) {
```

But please check it out.

Reply:
Yup, You are right.